tests/Unit/Rules/CombinedFieldsMaxRuleTest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Microservice\Tests\Unit\Rules;
+
+use Illuminate\Validation\Validator;
+use MyParcelCom\Microservice\Tests\TestCase;
+
+class CombinedFieldsMaxRuleTest extends TestCase
+{
+    /** @test */
+    public function testItPassesWhenTheCombinedFieldsDoNotExceedMaxCharacterLength()
+    {
+        $data = [
+            'fields' => [
+                'field-1' => 'is 21 characters long',
+                'field-2' => 'is 4',
+                'field-3' => 'is 9 long',
+            ],
+        ];
+
+        /** @var Validator $validator */
+        $validator = app()->make('validator')->make($data, [
```

Comment:
Isn't this the same as:
```suggestion
        $validator = app('validator')->make($data, [
```